ui/src/components/Nodes/NodeStructurePanel.vue

```diff
@@ -122,11 +122,11 @@ const onCategorySwitchClick = () => {
 }
 
 const onClearCategories = () => {
-  nodeStructureStore.setSelectedCategories([])
+  // nodeStructureStore.setSelectedCategories([])
```

Comment:
Should these commented-out lines be removed, or are they going to be implemented?